packages/v4/src/content/design-guidelines/components/card/card.md

```diff
@@ -3,116 +3,99 @@ id: Card
 
-Cards are flexible, and you can customize them in a variety of ways. A card can contain:
-- A header with an image, actions, and a body.
-- Only a header and a body. 
-- A header with an image, a body, and footer. 
+2. **Title(optional):** Communicates the title of the card if it's not included in the header.
 
-PatternFly supports two types of cards: 
-1. **Compact (recommended):** Compact cards use 16px spacing with an 8px spacer between the header and body and 16px between the body and footer.
-2. **Standard:** Standard cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.
+3. **Body:** Provides details about the item. The card body can include any combination of static text and/or active content.
 
-<img src="./img/card-types.png" alt="compact and standard card types" /> 
+4. **Footer(optional):** Contains links, actions, or static text at the bottom of a card.
 
-## Card usage
-Use a card:
-- To visually group content or distinguish information from the rest of a page’s content. Examples include dashboards and catalog views. 
-- To organize content in layouts with multiple columns or grids.  
-- To compare separate groups of information.  
-- To make media such as images and videos easier to scan.  
+Cards are flexible and customizable to your use case. Here are some different ways to use a card to represent the same content. Choose a layout that considers the usage context and the space you have to work with.
+
+<img src="./img/card-variants.png" alt="three card variants" width="1202"/>
 
-<img src="./img/dashboard.png" alt="card view with charts" /> 
-<br />
-<br />
-Do NOT use a card to show a dense amount of repetitive, related content. Use a list or table view instead.
+1. The first version places an image in the header with a separate title. The View details action is located in the footer.
 
+2. In the second verion, the title is moved to the header area to give a more compact footprint.
 
-## Card view usage
-A **card view** is a grid of cards that displays a small to moderate amount of content. 
+3. A third version removes the footer, adds the image back, and makes the title a link to drill into the details, which makes the footprint of the card smaller, yet.
 
-<img src="./img/card-view.png" alt="card view" /> 
+### Content spacing
 
-Use a card view to:
-- Display items in a data set that are best identified by a graphic or other visual representation.
-- Make information easier to visualize and compare. 
-- Summarize a large number of objects at once within the same page. 
+PatternFly supports three types of cards:
 
-Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+1. **Compact (recommended):** Use 16px spacing with an 8px spacer between the header and body, and a 16px spacer between the body and footer.
```

Comment:
Would change this to say "Compact (recommended): Compact cards use 16px spacing with...." to keep the structure of the sentences in this numbered list consistent